packages/accounts-base/accounts_server.js

```diff
@@ -399,19 +448,19 @@ Meteor.methods({
   //   If successful, returns {token: reconnectToken, id: userId}
   //   If unsuccessful (for example, if the user closed the oauth login popup),
   //     throws an error describing the reason
-  login: function(options) {
+  login: function (options) {
     var self = this;
 
     // Login handlers should really also check whatever field they look at in
     // options, but we don't enforce it.
     check(options, Object);
 
-    var result = runLoginHandlers(self, options);
+    var result = Accounts._runLoginHandlers(self, options);
```

Comment:
It's a shame that we have no access to a specific `AccountsServer` object in methods, so we have to continue assuming the default `Accounts` object.